app/processing/get-payment-requests-from-file.js

```diff
@@ -0,0 +1,11 @@
+const readline = require('readline')
+const { Readable } = require('stream')
+const getPaymentRequestsFromSitiAgriFile = require('./siti-agri/get-payment-requests')
+
+const getPaymentRequestsFromFile = (fileBuffer, scheme) => {
+  const input = Readable.from(fileBuffer)
+  const readBatchLines = readline.createInterface(input)
+  return getPaymentRequestsFromSitiAgriFile(readBatchLines, scheme, input)
```

Comment:
Deliberately structuring like this where we abstract all reading of files into specific source system modules.  Will allow extension to support other system batch files in the future with only a minor change here.